src/wizard/interactive_build.jl

```diff
@@ -184,6 +184,60 @@ function diff_srcdir(state::WizardState, prefix::Prefix, ur::Runner)
     return false
 end
 
+function bb_add(client, state::WizardState, prefix::Prefix, platform::Platform, jll::AbstractString)
+    if any(dep->getpkg(dep).name == jll, state.dependencies)
+        println(client, "ERROR: Package was already added")
+        return
+    end
+    new_dep = Dependency(jll)
+    try
+        # This will redo some work, but that may be ok
+        setup_dependencies(prefix, getpkg.([state.dependencies; new_dep]), platform)
+        push!(state.dependencies, new_dep)
+    catch e
+        showerror(client, e)
+    end
+end
+
+function setup_bb_service(state::WizardState, prefix, platform)
+    fpath = joinpath(prefix, "metadir", "bb_service")
+    server = listen(fpath)
+    @async begin
+        while isopen(server)
+            client = accept(server)
+            function client_error(err)
+                println(client, "ERROR: $err")
+                close(client)
+            end
+            @async while isopen(client)
+                try
+                    cmd = readline(client)
+                    ARGS = split(cmd, " ")
+                    length(ARGS) == 0 && client_error("Expected a command")
+                    if length(ARGS) >= 2 && ARGS[1] == "bb"
+                        if ARGS[2] == "add"
+                            if length(ARGS) == 2
+                                println(client, "Usage: bb add <jll>")
+                                close(client)
+                            end
+                            bb_add(client, state, prefix, platform, ARGS[3])
+                            close(client)
+                        else
+                            client_error("Unrecognized subcommand $(ARGS[2])")
+                        end
+                    else
```

Comment:
We should add a `bb`/`bb help`/`bb --help` that lists all options and examples.

Reply:
I guess at that point, we might as well just use ArgParse.jl directly?

Reply:
Sure